dev/tests/_bootstrap.php

```diff
@@ -45,7 +45,8 @@
     'MAGENTO_BACKEND_NAME' => 'admin',
     'MAGENTO_ADMIN_USERNAME' => 'admin',
     'MAGENTO_ADMIN_PASSWORD' => 'admin123',
-    'DEFAULT_TIMEZONE' => 'America/Los_Angeles'
+    'DEFAULT_TIMEZONE' => 'America/Los_Angeles',
+    'WAIT_TIMEOUT' => '10'
```

Comment:
Do we want this WAIT_TIMEOUT to be defined in .env ? or hiding from user in bootstrap?

Reply:
Never mind. WAIT_TIMEOUT is in .env.example.

Reply:
Let's add WAIT_TIMEOUT in Dev Doc.

Reply:
Yup, I'll add it to devdocs!